app/views/api/services/forms/_authentication_settings.html.slim

```diff
@@ -0,0 +1,40 @@
+= f.inputs class: 'auth-settings-wrapper' do
+  - # oidc
+  - is_oidc = @service.backend_version === 'oidc'
+  = f.inputs id: 'service_proxy_authentication_method_oidc_settings', disabled: !is_oidc, class: "auth-settings #{'hidden' unless is_oidc}"
+    = render 'api/services/forms/oidc', f: f, proxy: @service.proxy
+
+  - # auth user key (1)
+  - is_auth_user = @service.backend_version === '1'
```

Comment:
It should be a method to be called from `@service` and definitely not with the same `is_...`.
But I guess this is also copy-pasted :smile: